ouroboros-consensus-byron/src/Ouroboros/Consensus/Byron/Node/Serialisation.hs

```diff
@@ -148,7 +152,12 @@ instance SerialiseNodeToNode ByronBlock (GenTxId ByronBlock) where
   SerialiseNodeToClient
 -------------------------------------------------------------------------------}
 
-instance SerialiseNodeToClientConstraints ByronBlock
+instance SerialiseNodeToClient ByronBlock (LedgerConfig ByronBlock)
```

Comment:
Doesn't the `SerialiseNodeToClient ByronBlock Cardano.Chain.Genesis.Config` instance below provide this? Can we drop this new instance context element?

Reply:
yup